packages/calcite-components/src/components/input-date-picker/input-date-picker.tsx

```diff
@@ -689,6 +693,7 @@ export class InputDatePicker
           {this.validationMessage && this.status === "invalid" ? (
             <Validation
               icon={this.validationIcon}
+              id={IDS.validationMessage}
```

Comment:
Since IDs are scoped within their respective shadow DOM, one option worth exploring (separately) to simplify this pattern would be:

1. define a common validation message ID (exported from `Validation.tsx`)
2. always assign the common ID `Validation`'s `input-message` and optionally drop `id` from `ValidationProps` (could keep as optional to allow overrides if truly needed)
3. use the common ID in all `aria-errormessage` props

Reply:
@benelan Do you think we should add coverage for this in `formAssociated`? We could possibly assert on the validation ID being associated with `aria-errormessage` internally. 

If so, we can tackle this separately.

Reply:
@jcfranco I created this follow up [issue](https://github.com/Esri/calcite-design-system/issues/10082) to work on simplifying the validation message IDs.